packages/styles/src/components/tabs/_tab-title.scss

```
@@ -0,0 +1,153 @@
+@forward './../../variables/options';
+
+@use './../../mixins/color' as color-mx;
+@use './../../mixins/utilities' as utilities-mx;
+@use './../../variables/color';
+@use './../../variables/spacing';
+@use './../../variables/components/nav';
+
+.tab-title {
+  display: inline-block;
+  position: relative;
+  box-sizing: border-box;
+  padding: nav.$nav-link-padding;
+  transition: background-color 100ms;
+  border-right: 1px solid transparent;
+  border-left: 1px solid transparent;
+  outline-color: currentColor;
+  opacity: 0.7;
+  color: var(--post-contrast-color);
+  text-decoration: none;
+
+  &:focus {
+    opacity: 0.7;
```

Opacity is defined twice.

```suggestion
  opacity: 0.7;
  color: var(--post-contrast-color);
  text-decoration: none;

  &:focus {
```